src/01-globals/information/Information.vue

```diff
@@ -0,0 +1,18 @@
+<template src="./Information.html" />
+
+<script>
+import AHeading from '../../atoms/heading/Heading.vue'
+import AParagraph from '../../atoms/paragraph/Paragraph.vue'
+
+import Information from './Information.js'
```

Comment:
hmm maybe it would be better to add prefix also here  `AInformation` and the same proposal for other components imported like above.